src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -217,15 +217,14 @@ export const PaymentContent: FC<React.PropsWithChildren<Props>> = props => {
 const getAvailablePaymentMethods = (
   availablePaymentMethods: readonly CommercePaymentMethodEnum[],
 ): ReactElement<RadioProps<string>>[] => {
-  let paymentMethod: CommercePaymentMethodEnum = "CREDIT_CARD"
```

Comment:
Not sure why we needed this and I was getting warning about non ideal assignments below.